assets/javascripts/summary/index.js

```diff
@@ -1,9 +1,26 @@
-var map = require('../map.js')
+var ol = require('openlayers')
+var Map = require('../map.js')
 
 function Summary (options) {
   var easting = window.encodeURIComponent(options.easting)
   var northing = window.encodeURIComponent(options.northing)
-  map.loadMap(easting && [easting, northing], true)
+
+  var mapOptions = {
+    point: [parseInt(easting), parseInt(northing)],
+    layers: [
+      new ol.layer.Image({
+        ref: 'fmp',
+        source: new ol.source.ImageWMS({
+          url: '/wms-proxy',
+          serverType: 'geoserver',
+          params: {
+            'LAYERS': 'fmp'
+          }
+        })
+      })
+    ]
+  }
+  Map(mapOptions)
```

Comment:
Standard will then complain that the variable is unused, hence why it's that way